code/src/test/java/com/google/sps/data/BusinessTest.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+import org.junit.Test;
+import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+ 
+@RunWith(JUnit4.class)
+public final class BusinessTest {
+
```

Comment:
I see that this is testing that the contact method works but what I would do is have two seperate tests, for one assert false and do not call contact and for the other call contact and assert true. This would avoid any hiccups from having multiple assertions in one test

Reply:
Could call the first one createBusinessNoContact and the second createContactedBusiness

Reply:
I may be wrong tho, I'd ask Ben and Grady about testing

Reply:
I like the idea, makes it easier to debug, will implement rn